sphinx/domains/c.py

```diff
@@ -73,158 +2859,162 @@ class CObject(ObjectDescription):
               names=('rtype',)),
     ]
 
-    # These C types aren't described anywhere, so don't try to create
-    # a cross-reference to them
-    stopwords = {
-        'const', 'void', 'char', 'wchar_t', 'int', 'short',
-        'long', 'float', 'double', 'unsigned', 'signed', 'FILE',
-        'clock_t', 'time_t', 'ptrdiff_t', 'size_t', 'ssize_t',
-        'struct', '_Bool',
-    }
+    def warn(self, msg: Union[str, Exception]) -> None:
+        self.state_machine.reporter.warning(msg, line=self.lineno)
```

Comment:
Now we've used the `logger` library. It's simple and portable. So `warn()` methods are no longer needed. Please consider the replacement of them in the future.

Reply:
Ah, I didn't notice that change. I'll get it fixed in the near'ish future.